JSDemos/Demos/TreeList/Resizing/description.md

```diff
@@ -1 +1,24 @@
-The TreeList allows a&nbsp;user to&nbsp;resize columns at&nbsp;runtime. They can be&nbsp;resized in&nbsp;two different modes: by&nbsp;changing the width of&nbsp;the next column or&nbsp;the width of&nbsp;the component. To&nbsp;switch the current mode, use the &laquo;Column resizing mode&raquo; select box.
\ No newline at end of file
+TreeList automatically sets equal widths for all columns. You can make column widths adjust to the content, set custom widths, and allow users to resize columns.
```

Comment:
> automatically 

I read it as "TreeList sets equal widths for all columns no matter what you do".

> You can make column widths adjust to the content, set custom widths, and allow users to resize columns.

The subheadings tell for themselves, there's no need to recount them.


```suggestion
The TreeList sets equal widths for all columns, but you can change the widths as described below:
```